Source/WebCore/dom/TrustedType.cpp

```diff
@@ -170,6 +184,48 @@ ExceptionOr<String> trustedTypeCompliantString(TrustedType expectedType, ScriptE
     return stringValue;
 }
 
+String getAttributeTrustedType(const String& tagName, const String& attributeParameter, const String& elementNamespace, const String& attributeNamespace, String& sink)
+{
+    auto localName = tagName.convertToASCIILowercase();
+    auto attributeName = attributeParameter.convertToASCIILowercase();
+
+    if (attributeName.startsWith("on"_s))
+        return trustedTypeToString(TrustedType::TrustedScript);
```

Comment:
This path should set the sink value too. Not sure how best to calculate the value for this.

Reply:
Actually scratch that I don't think it needs to because it can only be hit via setAttribute. Rather than the IDL attribute path.